packages/repository/src/__tests__/unit/repositories/relations-helpers/relations-helpers-fixtures.ts

```diff
@@ -14,36 +14,89 @@ import {
   juggler,
   model,
   property,
+  hasOne,
+  HasOneRepositoryFactory,
 } from '../../../..';
 
+@model()
+export class Manufacturer extends Entity {
+  @property({id: true})
+  id: number;
+  @property()
+  name: string;
+  @belongsTo(() => Product)
+  productId: number;
+}
+interface ManufacturerRelations {
+  products?: Product;
+}
+
+export class ManufacturerRepository extends DefaultCrudRepository<
```

Comment:
I am not very happy that unit tests are calling relations, can we move such tests to `integration` folder? Sorry for not catching this issue earlier. If you prefer, I am ok to leave this change out of scope of this pull request, to get this building block landed sooner.

Reply:
These are just fixtures for test usage. But I will move them to `belongsTo`.